container/container.go

```diff
@@ -0,0 +1,27 @@
+package container
+
+import (
+	"context"
+	"strings"
+
+	crname "github.com/google/go-containerregistry/pkg/name"
+	"github.com/sigstore/cosign/pkg/cosign"
+	"github.com/sigstore/cosign/pkg/oci"
+)
+
+var GetImageDigest = func(imageReference string) (string, error) {
+	ref, err := crname.ParseReference(imageReference)
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimPrefix(ref.Context().Digest(ref.Identifier()).DigestStr(), "sha256:"), nil
+}
+
+var RunCosignImageVerification = func(ctx context.Context,
```

Comment:
Why not have this under `gha/`?

Reply:
Other verifiers may want it: If GCB is also signing containers, then I assume it will also want to run image verification, right?

Reply:
but GCB won't be using cosign's format, at least we don't know that. Their first version wont use it for sure.
I see what you mean, though. This would be useful for any builder that integrates with Sigstore / Fulcio via OIDC, correct?

Makes sense, let's keep it here then.